kie-remote/src/test/java/org/kie/remote/command/SerializableCommandConstructorTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.remote.command;
+
+import java.io.Serializable;
+
+import org.junit.Test;
+import org.kie.remote.impl.RemoteFactHandleImpl;
+
+import static org.junit.Assert.*;
+
+public class SerializableCommandConstructorTest {
```

Comment:
Please remove this class too, we should not be obsessed by SonarCloud coverage :)

Reply:
ok